CHANGELOG.md

```diff
@@ -10,7 +10,11 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ### Added
 
-* **gbasf2:** New setting `gbasf2_setup_path` which can be used to customize the path to the gbasf2 setup file directly (default: `"/cvmfs/belle.kek.jp/grid/gbasf2/pro/tools/setup.sh"`). It is a more flexible replacement for the `gbasf2_install_directory` setting, which will be removed in the future, since we can't predict potential name and path changes of the setup script between gbasf2 releases. @meliache #162
+* **gbasf2:** New setting `gbasf2_setup_path` which can be used to customize the path to the gbasf2 setup file directly (default: `"/cvmfs/belle.kek.jp/grid/gbasf2/pro/tools/setup.sh"`). It is a more flexible replacement for the `gbasf2_install_directory` setting, which will be removed in the future, since we can't predict potential name and path changes of the setup script between gbasf2 releases. @meliache [#162](https://github.com/nils-braun/b2luigi/pull/162)
+
+### Fixed
+
+- **gbasf2:** Fixed the issues caused by `gbasf2` release `v5r7` [#197](https://github.com/nils-braun/b2luigi/pull/197).
```

Comment:
Thanks for updating changelog. In this changelog I mostly use present tense for the bullet points, as is common for commit messages, so I'll suggest that for consistency (I googled what tense is commit for changelogs and it seems controversial, a matter of taste and consistency):
```suggestion
- **gbasf2:** Fix the issues caused by `gbasf2` release `v5r7` [#197](https://github.com/nils-braun/b2luigi/pull/197).
```